core/services/feeds/service.go

```diff
@@ -107,6 +107,9 @@ type Service interface {
 	ListSpecsByJobProposalIDs(ctx context.Context, ids []int64) ([]JobProposalSpec, error)
 	RejectSpec(ctx context.Context, id int64) error
 	UpdateSpecDefinition(ctx context.Context, id int64, spec string) error
+
+	// UnsafeSetConnectionsManager Only for testing
+	UnsafeSetConnectionsManager(ConnectionsManager)
```

Comment:
Chainlink code didn't have it. I re-added it manually. Didn't use the `_` because was getting some linting messages locally AFAIR. Does the underscore mean something in this context?

Reply:
It doesn't change the meaning of the func but just wondering if this'll botch up any future merges from ccip -> chainlink